getting_started/first_2d_game/04.creating_the_enemy.rst

```diff
@@ -24,9 +24,12 @@ Click Scene -> New Scene from the top menu and add the following nodes:
 Don't forget to set the children so they can't be selected, like you did with
 the Player scene.
 
-In the :ref:`RigidBody2D <class_RigidBody2D>` properties, set ``Gravity Scale``
-to ``0``, so the mob will not fall downward. In addition, under the
-:ref:`CollisionObject2D <class_CollisionObject2D>` section, uncheck the ``1`` inside the ``Mask`` property.
+Select the ``Mob`` node and set it's ``Gravity Scale`` property in the :ref:`RigidBody2D <class_RigidBody2D>`
+section of the inspector to ``0``, so the mob will not fall downward.
+
+In addition, under the :ref:`CollisionObject2D <class_CollisionObject2D>`
+section just beneath the **RigidBody2D** section,
+uncheck the ``1`` inside the ``Mask`` property.
```

Comment:
I don't think the Mask property is visible by default so it could be helpful to mention that too.

```
... just beneath the RigidBody2D section, expand Collision and uncheck the ``1`` inside the ``Mask`` property.
```